baselines/ts5.5/sharedworker.generated.d.ts

```diff
@@ -4208,8 +4217,6 @@ interface PerformanceResourceTiming extends PerformanceEntry {
     readonly requestStart: DOMHighResTimeStamp;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/PerformanceResourceTiming/responseEnd) */
     readonly responseEnd: DOMHighResTimeStamp;
```

Comment:
Add back incorrectly removed `responseStart`

```suggestion
    readonly responseEnd: DOMHighResTimeStamp;
    /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/PerformanceResourceTiming/responseStart) */
    readonly responseStart: DOMHighResTimeStamp;
```